app/overrides/o_data_controller_override.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+ODataController.class_eval do
+  before_action :refresh_schema
+
+  # The odata engine expects a static schema, but our schema may change
+  # whenever forms are updated and also depending on the current mission context.
+  def refresh_schema
+    schema = OData::ActiveRecordSchema::Base
+      .new("NEMO", skip_require: true,
+                   skip_add_entity_types: true,
+                   transform_json_for_root: ->(*args) { transform_json_for_root(*args) },
+                   transform_schema_for_metadata: ->(*args) { transform_schema_for_metadata(*args) },
```

Comment:
Do we need this?

Reply:
[ Yes, see <https://github.com/sassafrastech/odata_server/pull/2#discussion_r414903064> ]